test/test_integration_list.py

```diff
@@ -0,0 +1,324 @@
+# -*- coding: utf-8 -*-
+
+"""
+CLI List Integration tests
+
+This test file handles the main test flows for the list command
+"""
+import io
+from contextlib import redirect_stderr
+from unittest.mock import patch
+
+from .util.cli_helpers import integration_test
+from test.util.test_runners.integration_test_case import IntegrationTestCaseBase
+
+
+# Not coverage tested since integration tests doesn't run in
+# the same build step
+class CliIntegrationListTest(IntegrationTestCaseBase):  # pragma: no cover
+    DEFINED_VARIABLE_ID = 'one/password'
+    capture_stream = io.StringIO()
```

Comment:
using in only one test case. maybe it is better to use it only there. not healthy to start buffers for no reason. also better to close when finish uses.. also can use the with statement for this.

Reply:
I anticipated more but you are right. Will move to the scope of that specific test definition